src/codeflare_sdk/cluster/cluster.py

```diff
@@ -114,8 +114,10 @@ def up(self):
         """
         namespace = self.config.namespace
         try:
-            config.load_kube_config()
-            api_instance = client.CustomObjectsApi()
+            KubeConfigFileAuthentication.config_check()
+            api_instance = client.CustomObjectsApi(
+                TokenAuthentication.api_config_handler()
```

Comment:
Should this still be passed in if the config_check is successful?

Reply:
Yes because if `config_check` passes the result of `api_config_handler()` will be `None` which is the default functionality if a user has a loaded config file.